runtime/src/installed_scheduler_pool.rs

```diff
@@ -167,7 +167,8 @@ pub trait InstalledScheduler: Send + Sync + Debug + 'static {
     /// having &mut.
     fn schedule_execution<'a>(
         &'a self,
```

Comment:
i think `'a` can be elided, in turn remove some lint allow can be simplified like this:

```
allow(unused_attributes)
```

Reply:
a36489c7a610f258b01368c0d43ba0250bf3b5f6